accounts-geth/src/main/java/pm/gnosis/android/app/accounts/di/AccountsModule.kt

```diff
@@ -0,0 +1,23 @@
+package pm.gnosis.android.app.accounts.di
+
+import android.content.Context
+import dagger.Module
+import dagger.Provides
+import org.ethereum.geth.Geth
+import org.ethereum.geth.KeyStore
+import pm.gnosis.android.app.accounts.repositories.AccountsRepository
+import pm.gnosis.android.app.accounts.repositories.impl.GethAccountManager
+import pm.gnosis.android.app.accounts.repositories.impl.GethAccountsRepository
+import javax.inject.Singleton
+
+@Module
+class AccountsModule(val context: Context) {
```

Comment:
You can also use the `@ApplicationContext` to distinguish from a context annotated as `@ViewContext`